archive/forms.py

```diff
@@ -54,7 +70,7 @@ def save(self, *args, **kwargs):
         return collection
 
 class PublicAdminForm(forms.ModelForm):
-    files = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}),
+    files = forms.FileField(widget=forms.ClearableFileInput(),
```

Comment:
Was thinking for these to use multiple as well? `MultipleFileInput`

Reply:
This is for uploading public s3 files, used when making news posts with public images for example